www/src/pages/components/fab/ios/index.mdx

```diff
@@ -0,0 +1,137 @@
+---
+title: Fab
```

Comment:
nit: probably best to spell out "Floating Action Button" for the title

Reply:
The React one is also called "Fab" so probably better to leave this like this for now and change both in a follow-up PR if we'd like.